### PR TITLE
v4.2.4

### PR DESCRIPTION
**Pull request by frankmer**

### CODE REVIEW

File: lib/model/push_request_queue.dart [hunk 1]
```diff
@@ -53,6 +53,8 @@ class PushRequestQueue {
 
   PushRequest pop() => list.removeAt(0);
```
Comment (nilsbehlen) [inline lib/model/push_request_queue.dart:54]: why keep this if `tryPop` is safer?

File: lib/views/settings_view/settings_view.dart [hunk 2]
```diff
@@ -23,249 +24,252 @@ class SettingsView extends ConsumerWidget {
...
+                    // if (ref.watch(tokenProvider).hasHOTPTokens)
+                    //   ListTile(
+                    //     title: RichText(
+                    //       text: TextSpan(
+                    //         children: [
+                    //           TextSpan(
+                    //             text: AppLocalizations.of(context)!.hidePushTokens,
+                    //             style: Theme.of(context).textTheme.titleMedium,
+                    //           ),
+                    //         ],
```
Comment (nilsbehlen) [inline lib/views/settings_view/settings_view.dart:232]: is this still needed?

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

File: lib/model/push_request_queue.dart (hunk 1 above)
Comment (frankmer) [reply lib/model/push_request_queue.dart:54]: Is removed now.

File: lib/views/settings_view/settings_view.dart (hunk 2 above)
Comment (frankmer) [reply lib/views/settings_view/settings_view.dart:232]: Is removed now.

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**